content/doc/book/blueocean/activity.adoc

```diff
@@ -19,93 +19,72 @@ endif::[]
 
 = Activity View
 
-The Blue Ocean Activity View shows all activities related to one Pipeline.
-
-image:blueocean/activity/overview.png[Overview of the Activity View, role=center]
+The Blue Ocean Activity view shows all activities related to one Pipeline.
 
 include::doc/book/blueocean/_blue-ocean-status.adoc[]
 
 == Navigation Bar
 
-The Activity View includes the <<getting-started#navigation-bar, standard navigation bar>>
-at the top, with a local navigation bar below that.
+The Activity view includes the <<getting-started#navigation-bar, standard navigation bar>> at the top and a local navigation bar below that.
```

Comment:
The word "it" seems to read a little better for me, but you can take or leave this suggestion at your discretion. 
```suggestion
The Activity view includes the <<getting-started#navigation-bar, standard navigation bar>> at the top and a local navigation bar below it.
```